src/utils.rs

```diff
@@ -4,19 +4,6 @@
 
 use ffi;
 use translate::*;
-use UserDirectory;
-
-pub fn get_application_name() -> Option<String> {
```

Comment:
Hm, you right. I missed fact that auto non public in all crates :(
What better do: add to lib.rs `pub mod functions` with reexports or something else?
@GuillaumeGomez maybe you have better idea?